docs/Multiremote.md

```diff
@@ -30,7 +30,7 @@ import { multiremote } from 'webdriverio';
     await browser.url('http://json.org');
 
     // click on an element at the same time
-    const elem = await browser.$('#someElem');
+    const elem = await $('#someElem');
```

Comment:
Actually this example requires `browser.$` because it is not run with the wdio testrunner.